runtime/astar/src/lib.rs

```diff
@@ -1189,6 +1189,25 @@ impl frame_support::traits::SortedMembers<AccountId> for OracleMembershipWrapper
     }
 }
 
+parameter_types! {
+    pub MbmServiceWeight: Weight = Perbill::from_percent(80) * RuntimeBlockWeights::get().max_block;
+}
```

Comment:
One concern, but maybe it's not as important - does this assume that nothing else can happen?

Because e.g. democracy can consume essentially the entire block weight (not for real but it will consume all when weight tracking is concerned), and we also have dApp staking era change calculations which can consume significant amount of block space.

What would happen if e.g. migration consumes 80% of the block weight, and era change needs to consume 30%?

Reply:
era change is handled on_initialize and this 80% is for block_weight which itself is 75%

Reply:
I meant 80% of normal weight

Reply:
@Dinonard updated to 50%